.github/workflows/release.yaml

```
@@ -0,0 +1,34 @@
+name: release main
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "tools/*"
+      - ".github/workflows/release.yaml"
+
+jobs:
+  release:
+    env:
+      RELEASE_FILE_NAME: tools.tar.gz
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Package Scripts
+        run: |
+          tar -czf ${{ env.RELEASE_FILE_NAME }} -C ./tools .
+      - name: Compute Next Tag Name
+        run: |
+          LAST_TAG=$(git describe --tags --abbrev=0)
```

```suggestion
          LAST_TAG=$(git describe --tags --abbrev=0 || echo "0.0.0")
```

I know this was copied from the other repo. I think it will fail since no tags currently exist in the repository. We might want to add an `||` statement to account for this.